Header header
int16 count # 当前消息所包含检测目标数量
int16 camera_status # 摄像头是否故障，0故障，1正常
BoundingBox[] bounding_boxes